ads_plugin.py

```diff
@@ -36,6 +34,8 @@
 
 from AxonDeepSeg.morphometrics.compute_morphometrics import *
 
+VERSION = "0.2.2"
```

Comment:
I would specify in the name that it's the plugin version, to avoid confusion with the ADS version. so maybe rename it to PLUGIN_VERSION